src/app/card/card.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, Input, Output, EventEmitter, OnDestroy } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { Broadcaster, Logger } from 'ngx-base';
+
+import { WorkItem } from './../models/work-item';
+
+@Component({
+  selector: 'card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss']
+})
+export class CardComponent implements OnInit {
+
+  @Input() item: WorkItem;
```

Comment:
@vikram-raj Well it maybe indivisible or separate input. Doesn't really matter, but it has to be generic.